c10/util/Float4_e2m1fn_x2.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <cstdint>
+
+#include <c10/macros/Macros.h>
+
+/// Defines the Float4_e2m1fn_x2 type (4-bit floating-point, two elements packed
+/// into one byte) Binary configuration :
+///
+///   0123:4567
+///   seem:seem
```

Comment:
It's a bit confusing the notation, as I assume that to get the `i`-th bit as defined here you should do `(v >> (7 - i)) & 1`...

Also, if I do `v & 0xF` do I get the first element or the second element of the tensor?